learn/DOM/009_Nodelist/script.js

```diff
@@ -0,0 +1,20 @@
+const nodeList = document.querySelectorAll("p");
+const test = document.getElementById("test");
+const check = document.querySelectorAll("input[type=checkbox]");
+
+// Mengambil elemen nodelist menggunakan index
+test.innerHTML =
+  "Saya berkata :" + nodeList[0].innerHTML + "&" + nodeList[1].innerHTML;
```

Comment:
Bagian yang ini diubah aja, append text node, terus jadiin template literal

```suggestion
const text = document.createTextNode(`Saya berkata : ${nodeList[0].innerHTML} & ${nodeList[1].innerHTML}`);
test.appendChild(text);
```